### ISSUE TITLE
keeps asking for password in Debian/Thunderbird 52.9.1

### ISSUE THREAD

**Issue by garciafa** [vanowm/MasterPasswordPlus#141]
Master Pasword + keeps asking for the master password when I start up thunderbird and it does not hide the window.

This happened this morning for the first time.

Best regards,
Fabien Garcia

**Comment by whimboo** [comment]
I can second this for a Thunderbird installation of a friend, who also has this add-on installed. There are some overlay problems with Thunderbird (German locale) when opening the preferences. Attached are the [logs](https://github.com/vanowm/MasterPasswordPlus/files/2290883/trace.log).

I for myself work on OS X and have the extension installed too. But it works just fine with en-US.

**Comment by vanowm** [comment]
@garciafa please provide messages from console log. Preferably from the top after TB restart.

@whimboo there must be some more errors before messages you showed.

I can only test myself on Windows OS, so bare with me.

If you could try the following:
1) exit TB and create a copy of your profile (in case something goes wrong)
2) uninstall MP+ from Thunderbird's addon manager
3) restart TB
4) install MP+ again (from addon manager)

If this helped, then there must be something wrong with TB not clearing cache after installing new/update an extension.

**Comment by garciafa** [comment]
I have completely uninstalled MP+ (so much that I actually erased all my TB settings by mistake and took over with a clean .thunderbird directory :p ) and restarted TB. The problem is still there. 

Here is the log [Log_TB.txt](https://github.com/vanowm/MasterPasswordPlus/files/2294091/Log_TB.txt)

I tried one password entry but it outputs an error "cancelled by user" in the console.

Thanks for your help !
